compiler/qsc_parse/src/expr/tests.rs

```diff
@@ -342,6 +342,7 @@ fn lit_double_trailing_exp_dot() {
     );
 }
 
+#[ignore = "This test doesn't make sense."] // ToDo: Fix this test.
 #[test]
 fn lit_int_hexadecimal_dot() {
     check(expr, "0x123.45", &expect!["Expr _id_ [0-5]: Lit: Int(291)"]);
```

Comment:
This test doesn't make sense to me. In Q#, you can't have `0x` as a preface to a double, but even if you could, this test expects that it would be an Int instead. Can we remove this test?

Reply:
This is just testing to make sure the parser doesn't misinterpret this number. If you put it into the playground, you see whole behavior: `0x123` is correctly parsed as the integer 291 and then the `.` is treated as a syntax error. Effectively this is making sure that a hex number is interpreted as an integer even when it has a dot after it, and that the dot doesn't "steal" any digits from the hex number (as in, `0x123.45` somehow becoming `0x12` followed by `3.45`, which would be wrong). I think the test is worth keeping as it shows that digits followed by dot are interpreted correctly, and if the test isn't working it's worth digging in to understand why.

Reply:
The test won't work the way it did before because the parser will now parse the first part of it as an expression, and the `.` as a path/field accessor operator. It isn't until the type-checking that the expression can be identified as wrong.